subprocess/path.go

```diff
@@ -52,6 +53,11 @@ func LookPath(file string) (string, error) {
 	path := os.Getenv("PATH")
 	for _, dir := range filepath.SplitList(path) {
 		if dir == "" {
+			// Windows often has empty components in the PATH and
+			// treating them as "." not expected.
```

Comment:
Very minor nit: maybe revise to `treating them as "." is not expected` (i.e., add an "is")?

Reply:
Fixed.